inc/enum.h

```diff
@@ -27,6 +27,14 @@ enum LogThreshold {
 	LOG_THRESHOLD_DEFAULT = INFO,
 };
 
+enum LayoutFormatElem {
+	LAYOUT = 'l',
+	COUNT = 'c',
+	RATIO = 'r',
```

Comment:
Can we please add `{n}` for name i.e. wide, left etc. ?

This will allow users to extract the current layout, as the image is variable.